### PR TITLE
[improve][doc] Add description for using Copper Argos

### PR DESCRIPTION
**Pull request by massakam**
### Documentation

In https://github.com/apache/pulsar/pull/19445, I added some new parameters to the authentication plugin for Athenz, so I added a description about them.

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by momo-jun** [comment]
@massakam thanks for adding the docs!
I pushed a commit to fix a display issue of code blocks, which I found when I was reviewing the content. No offence:) It's not related to your additions.

File: docs/security-athenz.md [hunk 1]
```diff
@@ -172,6 +177,45 @@ client, err := pulsarNewClient(ClientOptions{
 </Tabs>
 ````
 
+### When using Copper Argos
```
Comment (momo-jun) [inline docs/security-athenz.md:180]:
```suggestion
### Use Copper Argos
```

Comment (massakam) [reply docs/security-athenz.md]: Fixed.

**Comment by Anonymitaet** [comment]
ping @nodece

**Comment by momo-jun** [comment]
@nodece Can you pls review this PR from the technical perspective?

**Review by nodece — APPROVED** [review]

**Comment by nodece** [comment]
@momo-jun @Anonymitaet Done.

**Pull request merged**